dist/less/overlay-panel.less

```diff
@@ -89,6 +103,9 @@ body.overlay-open {
     height: @wizard-pf-steps-indicator-height;
     margin: @wizard-pf-steps-indicator-margin-top-bottom 0;
     padding: 0;
+    @media(max-width: @screen-sm-min) {
+      margin-top: calc(@wizard-pf-steps-indicator-margin-top-bottom + 10px);
```

Comment:
Rather than adding more margin to accommodate the close button (which is why I assume you're doing this), how about we move the close closer to the edges of the modal, thus saving valuable vertical screen real estate?

```
.catalogs-overlay-panel-close {
  @media(max-width: @screen-sm-min) {
    right: 15px;
    top: 10px;
  }
}
```

looks like it works.

![screen shot 2017-05-17 at 9 58 44 am](https://cloud.githubusercontent.com/assets/895728/26157599/a3e771b6-3ae7-11e7-9d5c-2518f1b197c5.PNG)